torch/lib/THD/base/data_channels/DataChannelGloo.cpp

```diff
@@ -40,7 +66,11 @@ DataChannelGloo::~DataChannelGloo() {
 
 
 bool DataChannelGloo::init() {
-  getFullMeshCtx(); // wait for everyone to connect
+  for (uint8_t i = 0; i < (uint8_t)data_operation::last; i++) {
```

Comment:
http://en.cppreference.com/w/cpp/language/enum#Scoped_enumerations see example. If C++ documentation uses this as example I think we should trust them :D.

btw. it should be `static_cast<uint8_t>(data_operation::last)`